tests/sentry/api/endpoints/test_organization_dashboard_details.py

```diff
@@ -36,20 +36,26 @@ def setUp(self):
             widget=self.widget_1,
             name=self.anon_users_query["name"],
             fields=self.anon_users_query["fields"],
+            columns=self.anon_users_query["columns"],
+            aggregates=self.anon_users_query["aggregates"],
             conditions=self.anon_users_query["conditions"],
             order=0,
         )
         self.widget_1_data_2 = DashboardWidgetQuery.objects.create(
             widget=self.widget_1,
             name=self.known_users_query["name"],
             fields=self.known_users_query["fields"],
+            columns=self.anon_users_query["columns"],
```

Comment:
should we have backend changes in a separate PR?

Reply:
yup, I have a backend PR open, can you take a look? https://github.com/getsentry/sentry/pull/32518 @priscilawebdev